### ISSUE TITLE
Source tree does not maintain correct sorting when navigating to different pages

### ISSUE THREAD

**Issue by codehag** [devtools-html/debugger.html#3731]
When loading the debugger for a page the first time, the debugger source tree has the correct sorting, however if you navigate away from that page to another one, the source tree will be out of order. It looks like `this.props.debuggerUrl` never changes

Screenshot of correct sorting (when using an initial load for google.com):
<img width="251" alt="screen shot 2017-08-31 at 2 13 29 pm" src="https://user-images.githubusercontent.com/26968615/29943561-21b5a956-8e57-11e7-8c37-b34e6adde470.png">

Screenshot of incorrect sorting (when navigating to google.com from the debuggee):
<img width="249" alt="screen shot 2017-08-31 at 2 14 31 pm" src="https://user-images.githubusercontent.com/26968615/29943573-2feea8f6-8e57-11e7-893d-a7b75b37642e.png">

Screencap: http://g.recordit.co/kLBHdRJGlm.gif

**Comment by codehag** [comment]
Screenshot of correct sorting (when using an initial load for google.com):
<img width="251" alt="screen shot 2017-08-31 at 2 13 29 pm" src="https://user-images.githubusercontent.com/26968615/29943561-21b5a956-8e57-11e7-8c37-b34e6adde470.png">

Screenshot of incorrect sorting (when navigating to google.com from the debuggee):
<img width="249" alt="screen shot 2017-08-31 at 2 14 31 pm" src="https://user-images.githubusercontent.com/26968615/29943573-2feea8f6-8e57-11e7-893d-a7b75b37642e.png">

Screencap: http://g.recordit.co/kLBHdRJGlm.gif

**Comment by codehag** [comment]
this is probably the fix
```diff
diff --git a/src/reducers/pause.js b/src/reducers/pause.js
index 9615ead9..d003ee9f 100644
--- a/src/reducers/pause.js
+++ b/src/reducers/pause.js
@@ -145,6 +145,10 @@ function update(state: PauseState = State(), action: Action): PauseState {
 
     case "CLEAR_COMMAND":
       return { ...state, command: "" };
+
+    case "NAVIGATE":
+      return { ...state, debuggeeUrl: action.url };
+
   }
 
   return state;

```

**Comment by codehag** [comment]
fixed via #3882
